### PR TITLE
Clarify that docker plugin can accept multiple tags

### PR DESCRIPTION
**Pull request by scubbo**
See [forum
post](https://community.harness.io/t/possible-error-in-drone-docker-plugin-documentation/13434).

I used the Go syntax for collections types since Go looks like Drone's primary language.

It would be nice to have multiple examples (showing that `tags` can be both a single string or an array), but from a look at `scripts/buildPluginJson.mjs` it looks like the system only supports a single `example`, and that's probably a more-significant change than should be submitted without discussion.

There are probably _plenty_ of other plugin-variables which have a similar signature, but I suspect we can't easily search for that from this repo alone.

### CODE REVIEW

File: plugin-validation/plugin-schema.json [hunk 1]
```diff
@@ -86,7 +86,8 @@
                 "number",
                 "array",
                 "boolean",
-                "object"
+                "object",
+                "string or []string"
```
Comment (d1wilko) [inline plugin-validation/plugin-schema.json:90]: I think we can remove this - it will be covered by `array`

File: plugins/docker/content.yaml [hunk 2]
```diff
@@ -51,9 +53,9 @@ properties:
     secret: false
     required: false
   tags:
-    type: string
+    type: "string or []string"
```
Comment (d1wilko) [inline plugins/docker/content.yaml:56]: let's just make this `array` for now

**Review by d1wilko — CHANGES_REQUESTED** [review]
Hey, thanks for the PR :)

Can you remove the `package-lock.json` as you haven't made any changes to the `package.json`

I think it's ok if we change the type to `array` here for simplicity - even if the array only has one element - I'm hesitant to add too many enums for type

it does open up a bigger consideration about arrays having types... - but I think it needs to be clean, not simply adding more enums to the schema

I'll give it some thought :)

File: plugins/docker/content.yaml (hunk 2 above)
Comment (scubbo) [reply plugins/docker/content.yaml:56]: Sure thing, will do!

**Comment by scubbo** [comment]
Sorry for the delay!

**Review by d1wilko — APPROVED** [review]
LGTM - thank you :)

**Pull request merged**